aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -812,8 +897,7 @@ module aptos_framework::account {
 
     #[test_only]
     public fun get_signer_capability_offer_proof_challenge_v2(
-        source_address: address,
-        recipient_address: address,
+        source_address: address, recipient_address: address,
```

Comment:
Prefer for this not to happen

Reply:
It's actually quite difficult to make a choice here. One could be more radical, turning all function headers with more than one parameter into multi-line mode, with one parameter per line.